projects/storefrontapp-e2e-cypress/cypress/helpers/infinite-scroll.ts

```diff
@@ -104,35 +112,37 @@ export function scrollToFooter(
 }
 
 export function verifySortingResetsList() {
+  cy.route(
```

Comment:
not necessary like the comment above/below, but maybe we can pull this out to put in a function and call it as well in the top level of the `it`